packages/ai-react/tests/mcp-apps.test.tsx

```
@@ -0,0 +1,232 @@
+// @vitest-environment jsdom
+import type { UIResourcePart } from '@tanstack/ai'
```

ESLint

[error] `@tanstack/ai` type import should occur after import of `../src/mcp-apps`

(import/order)